src/pages/GenericHookGuides.mdx

```diff
@@ -0,0 +1,51 @@
+## Generic Guides
```

Comment:
Do you need this text here? This file only has one example

Reply:
I think in future there will be more generic topics which we can add here